CFP.md

```diff
@@ -0,0 +1,63 @@
+CALL FOR PAPERS
+====
+
+--------------------------------------------------------------------------------
+*Workshop for NLP Open Source Software (NLP-OSS)*
+20 July 2018, Co-located with ACL 2018
+https://nlposs.github.io/
+
+Deadline for Long and Short Paper submission: 25th March 2018 
+(23:59, GMT-11)
+---------------------------------------------------------------------------------
+
+The First Workshop for NLP Open Source Software (NLP-OSS) will be co-located 
+with ACL 2018 at Melbourne, Australia on 20 July 2018. 
+
+The Workshop for NLP-OSS hopes to be the intellectual forum to collate engineering 
+knowledge focusing more on the building aspect of NLP and less on scientific 
+novelty or state-of-art development. 
+
+NLP-OSS also provides an the academic workshop platforms to announce new software/features, 
+promote the open source culture and best practices that go beyond the conferences.
+
+We invite full papers (8 pages) or short papers (4 pages) on topics related to NLP-OSS 
+broadly categorized into (i) software development, (ii) scientific contribution 
+and (iii) NLP-OSS case studies.
+
+ - **Software Development**
+   - Designing and developing NLP-OSS
+   - Licensing issues in NLP-OSS
+   - Backwards compatibility and stale code in NLP-OSS
+   - Growing an NLP-OSS community
+   - Maintaining and motivating an NLP-OSS community
+   - Best practices for NLP-OSS documentation and testing
+   - Contribution to NLP-OSS without coding
+   - Incentivizing OSS contributions in NLP
+
+ - **Scientific Contribution**
+   - Benchmarking OSS for specific NLP task(s)
+   - Demonstration and tutorial of NLP-OSS
+   - New NLP-OSS introductions
+   - Small but useful NLP-OSS
+   - Machine learning vs NLP-OSS
+   - Citations and references for NLP-OSS
+   - OSS vs experiment replicability
+   - Gaps between existing NLP-OSS
+   - Task-independent NLP-OSS
+
+ - **Case studies**
+   - Case studies of how a specific bug is fixed or feature is added
+   - Writing wrappers for other NLP-OSS
+   - Writing open-source APIs for open data
+   - Teaching NLP with OSS
+   - Avoiding the hammer OSS in NLP
```

Comment:
I don't know what this means

Reply:
Using one tool to do anything. I think this is just a personal grudge I have against "everything can be solved by library X" and "if library X don't do task Y, use hack Z to make library X do task Y".

Maybe we should remove this too.

Reply:
Sounds like "task-generic vs task-specific", or rather the misuse of task-specific tools

Reply:
Lets remove this.